lookout/style/format/code_generator.py

```diff
@@ -170,7 +172,7 @@ def generate(self, vnodes: Sequence[VirtualNode], indentation: str) -> str:
 
         return "".join(tokens)
 
-    def _can_set_label(self, y_new: Sequence[int], y_old: Sequence[int],
+    def _can_set_label(self, y_new: Optional[Tuple[int, ...]], y_old: Tuple[int, ...],
```

Comment:
So we should write `Optional[Sequence[int]]` here. We do:

- `y_new is None`, hence `Optional`
- `y_new[0]`, hence `Sequence`
- Iterate `y_new`

Reply:
We have this annotation all over the code and actually want an immutable data structure for this attribute so I think Tuple is fine. We could also define `ClassSequence = Optional[Tuple[int, ...]]` and use it everywhere.

Reply:
This is not about what we want. This is about describing how the actual code works. The actual code does not care if `y_new` is mutable or immutable. I listed the only three operations our code performs here, and none of the three requires an immutable object.